salt/haproxy/haproxy.cfg.jinja

```diff
@@ -31,6 +31,8 @@ listen kubernetes-master
 {% endfor -%}
 
 {%- if "admin" in salt['grains.get']('roles', []) %}
+errorfile 503 /etc/caasp/haproxy/errors/503-velum_database_not_ready.http
```

Comment:
We also need to change caasp-container-manifests for this to be loaded on the very first run of the admin node.